### ISSUE TITLE
关于classification代码中一处意味不明的小错误

### ISSUE THREAD

**Issue by doc-ycm** [DAMO-DI-ML/NeurIPS2023-One-Fits-All#29]
在`class gpt4ts(nn.Module):`类中，有以下embedding模块定义
`self.enc_embedding = DataEmbedding(self.feat_dim * self.patch_size, config['d_model'], config['dropout'])`
该定义声明直接导致类对象生成时

```
class DataEmbedding(nn.Module):
    def __init__(self, c_in, d_model, embed_type='fixed', freq='h', dropout=0.1):
        super(DataEmbedding, self).__init__()
        self.value_embedding = TokenEmbedding(c_in=c_in, d_model=d_model)
        self.position_embedding = PositionalEmbedding(d_model=d_model)
        self.temporal_embedding = TemporalEmbedding(d_model=d_model, embed_type=embed_type,
                                                    freq=freq) if embed_type != 'timeF' else TimeFeatureEmbedding(
            d_model=d_model, embed_type=embed_type, freq=freq)
```
该形参embed_type值为config['dropout']，即错误赋值，其变成了一个float对象

进一步导致了self.temporal_embedding定义逻辑出现问题

但是由于在模型在forward函数中
        `outputs = self.enc_embedding(input_x, None)`
第二个形参为None，因此并未对实际运行逻辑产生影响

**Comment by PSacfc** [comment]
非常感谢，我们会对这个进行修改的！
